index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Ellon Mordecai</title>
+  </head>
+  <body>
+    <section id="nav-wrap">
+      <nav id="nav">
+        <h4 id="logo-placeholder">Welcome</h4>
+        <div id="menu-button-wrap">
+          <img src="https://img.icons8.com/ios-glyphs/30/172B4D/menu--v1.png"/>
+        </div>
+      </nav>
+    </section>
+    <main>
+      <header>
+        <h1 id="main-heading">
+            Hey there. I'm Ellon <br>
+            <span>I'm a software developer</span>
+        </h1>
+      </header>
+      <section id="supporting-text">
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don't hestiate to contact me.
+        </p>
+      </section>
+      <section id="social-icons">
+        <a href="#">
+          <span class="icon">
+            <img src="https://img.icons8.com/windows/32/172B4D/github.png"/>
+          </span>
+        </a>
+        <a href="#">
```

Comment:
- [ ] In this case, you're describing a list of social media icons, so it would be great if you used a `<ul>` tag which is perfect to describe a list. 🤓 Remember that providing semantic elements is very important for those people that rely on assistive technologies such as screen readers. Let's make the web more accessible! Please change this. 🛠️ 
- [ ] **[OPTIONAL]:** Kudos if you include your social media links inside the `href` property of your links! It would make it easier for people to connect with you! 🤓

Reply:
Got it. Thank you